src/lib/plot/series/bar-series.js

```diff
@@ -83,33 +83,36 @@ class BarSeries extends AbstractSeries {
   _updateSeries() {
     const container = getDOMNode(this.refs.container);
     const {
-      sameTypeTotal = 1,
-      sameTypeIndex = 0,
-      data} = this.props;
-
-    const {
+      _stackBy,
+      data,
       lineSizeAttr,
       valuePosAttr,
       linePosAttr,
       valueSizeAttr} = this.props;
 
+    let {
+      sameTypeTotal = 1,
+      sameTypeIndex = 0} = this.props;
+
     if (!data || !data.length) {
       return;
     }
 
     const distance = this._getScaleDistance(linePosAttr);
-    const baseValue = this._getScaleBaseValue(valuePosAttr);
     const lineFunctor = this._getAttributeFunctor(linePosAttr);
     const valueFunctor = this._getAttributeFunctor(valuePosAttr);
+    const value0Functor = this._getAttr0Functor(valuePosAttr);
+
+    if (_stackBy === valuePosAttr) {
+      sameTypeTotal = 1;
+      sameTypeIndex = 0;
```

Comment:
I'm guessing this means that you can't have multiple stacked column series like: 
![image](https://cloud.githubusercontent.com/assets/132584/13967078/a1c5e9fe-f032-11e5-83fc-f921003ada7e.png) (http://excelhelphq.com/tutorial-clustered-stacked-column-bar-charts-free/)